cmd/config.go

```diff
@@ -96,13 +97,18 @@ func newConfigCommand(globalConfig *settings.Config) *cobra.Command {
 			if len(args) == 1 {
 				path = args[0]
 			}
-			return compiler.ProcessConfig(config.ProcessConfigOpts{
```

Comment:
hmmm. Gotta revert this additional print statement. It's a big deal in a CLI.

Reply:
The ProcessConfig function was printing the same data inside the function. The function is now modified to return the data, and then printed. This change is done so that same function can be used in policy commands.